test/conftest.py

```diff
@@ -9,10 +9,18 @@
 import pytest
 
 
-@pytest.fixture(name="expected_hash")
-def fixture_expected_hash(request):
+@pytest.fixture(name="expected_tx_hash")
+def fixture_expected_tx_hash(request):
     """
-    Fixture to return values of expected hash
+    Fixture to return values of expected tx hash
+    """
+    return request.param
+
+
+@pytest.fixture(name="expected_block_hash")
+def fixture_expected_block_hash(request):
```

Comment:
Is this used anywhere? If these are used in just one file, they can be defined there instead of in conftest

Reply:
I was also planning to use this in test_block_number_incremented and test_deploy_lite but in the future.

You know version 2.0 (but for real not kidding).